### ISSUE TITLE
TypeError: test.wrap is not a function

### ISSUE THREAD

**Issue by frozenL** [firebase/functions-samples#358]
Hi,

I was trying to write offline unit testing code and this error occurred:
```
> mocha --reporter spec
  Cloud Functions
    compugeGraph
      1) processed all elements

  0 passing (28ms)
  1 failing

  1) Cloud Functions
       compugeGraph
         processed all elements:
     TypeError: test.wrap is not a function
      at Context.it (test/index.test.js:25:28)

npm ERR! Test failed.  See above for more details.

```
My package.json is:
```
{
  "name": "functions",
  "description": "Cloud Functions for Firebase",
  "scripts": {
    "lint": "eslint --fix . && eslint .",
    "serve": "firebase serve --only functions",
    "shell": "firebase experimental:functions:shell",
    "start": "npm run shell",
    "deploy": "firebase deploy --only functions",
    "logs": "firebase functions:log",
    "test": "mocha --reporter spec"
  },
  "dependencies": {
    "@google-cloud/storage": "^1.6.0",
    "firebase-admin": "^5.11.0",
    "firebase-functions": "^1.0.0",
    "line-by-line": "^0.1.6"
  },
  "devDependencies": {
    "eslint": "^4.12.0",
    "eslint-plugin-promise": "^3.6.0",
    "firebase-functions-test": "^0.1.0",
    "chai": "^3.5.0",
    "chai-as-promised": "^6.0.0",
    "mocha": "^5.0.5",
    "sinon": "^4.1.3"
  },
  "private": true
}
```
and my testing code is:
```
const chai = require('chai');
const assert = chai.assert;

const sinon = require('sinon');
const admin = require('firebase-admin');
const test = require('firebase-functions-test')();

describe('Cloud Functions', () => {
  const adminInitStub = sinon.stub(admin, 'initializeApp');
  const myFunctions = require('../index');
  const computeGraph = require('../computeGraph');

  before(() => {
  });

  after(() => {
    adminInitStub.restore();
    test.cleanup();
  });

  describe('compugeGraph', () => {
    it('processed all elements', () => {
      const data = require('./computeGraphDataIn.json');
      console.log(test);
      const wrapped = test.wrap(computeGraph.computeGraph);
      return assert.equal(wrapped(data).length, 69);
    })
  });
});
```
I wanted to see what 'test' itself was like, so I printed it:
```
{ features: 
   { mockConfig: [Function: mockConfig],
     wrap: [Function: wrap],
     makeChange: [Function: makeChange],
     analytics: 
      { makeAnalyticsEvent: [Function: makeAnalyticsEvent],
        exampleAnalyticsEvent: [Function: exampleAnalyticsEvent] },
     auth: 
      { makeUserRecord: [Function: makeUserRecord],
        exampleUserRecord: [Function: exampleUserRecord] },
     crashlytics: 
      { makeIssue: [Function: makeIssue],
        exampleIssue: [Function: exampleIssue] },
     database: 
      { makeDataSnapshot: [Function: makeDataSnapshot],
        exampleDataSnapshot: [Function: exampleDataSnapshot],
        exampleDataSnapshotChange: [Function: exampleDataSnapshotChange] },
     firestore: 
      { makeDocumentSnapshot: [Function: makeDocumentSnapshot],
        exampleDocumentSnapshot: [Function: exampleDocumentSnapshot],
        exampleDocumentSnapshotChange: [Function: exampleDocumentSnapshotChange],
        objectToValueProto: [Function: objectToValueProto] },
     pubsub: 
      { makeMessage: [Function: makeMessage],
        exampleMessage: [Function: exampleMessage] },
     storage: 
      { makeObjectMetadata: [Function: makeObjectMetadata],
        exampleObjectMetadata: [Function: exampleObjectMetadata] } },
  cleanup: [Function: cleanup] }
```

I'm new to Firebase and I didn't find anything relevant to this problem online. Could anyone plz help me with that?

Thanks a lot :)

**Comment by lingolive-tech** [comment]
I'm seeing this too, but was able to work around it for now using the following (from the above code):

`test.features.wrap(computeGraph.computeGraph)`

Is that an error in the examples or in the `firebase-functions-test` package?

**Comment by Westbrook** [comment]
I'm seeing this too, but was able to work around it for now using the following (from the above code):

test.features.wrap(computeGraph.computeGraph)

Is that an error in the examples or in the firebase-functions-test package?

**Comment by prescottprue** [comment]
I believe this is the same issue [as posted here on the `firebase-functions-test` package](https://github.com/firebase/firebase-functions-test/issues/2).

**Comment by nicolasgarnier** [comment]
Let's wait for @laurenzlong to comment on the issue :)

**Comment by laurenzlong** [comment]
Thanks for the quick feedback! Just released firebase-functions-test v0.1.1, which fixes this. The sample still fails because there's a bug in the index.js code, and it's also out of sync with the test code, but we'll get it fixed soon.
